microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/MicroProfileDefinition.java

```diff
@@ -173,6 +200,24 @@ private static MicroProfilePropertyDefinitionParams getPropertyDefinitionParams(
 		return definitionParams;
 	}
 
+	private static Either<List<? extends Location>, List<? extends LocationLink>>
+			getPropertyDefinition(PropertiesModel document, Property referencedProperty, PropertyValueExpression propertyValueExpression, boolean definitionLinkSupport) {
+		try {
+			Node key = referencedProperty.getKey();
+			Range referencedPropertyRange = new Range(document.positionAt(key.getStart()), document.positionAt(key.getEnd()));

```

Comment:
There is a helper function that can do this; instead of passing `document`, use the helper function.

Reply:
After looking at it, its still more practical to also pass `document`.